volcano.tf

```diff
@@ -0,0 +1,66 @@
+################################################################################
+# Volcano Scheduler for TorchX
+################################################################################
+
+resource "helm_release" "volcano" {
+  count = var.enable_volcano ? 1 : 0
+
+  name                       = try(var.volcano_helm_config["name"], "volcano")
```

Comment:
create a local variable for volcano and substitute in `name` ,  `chart`,  `namespace` as default value

```hcl
local {
  name = "volcano
}
```

Reply:
Done also for namespace as I followed this link's naming: https://volcano.sh/en/docs/installation/